modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPRequest.testcase

```diff
@@ -0,0 +1,234 @@
+			for (var userApproval : list "Manager,Finance") {
+				User.logoutAndLoginPG(
+					userLoginEmailAddress = "${userApproval}@liferay.com",
+					userLoginFullName = "${userApproval} User");
+
+				EVPUtils.gotoEVP();
+
+				EVPNavigator.goToNavItem(navItem = "Request");
+
+				EVPRequest.goToRequestOption(
+					key_option = "Review",
+					organizationName = "Organization Test");
+
+				EVPOrganization.changeStatus(
+					button = "Approve",
+					typeMessage = "Your organization has been approved!");
+			}
+		}
+
+		task ("When the Finance user do the Payment Confirmation") {
+			EVPRequest.goToRequestOption(
+				key_option = "Pay",
+				organizationName = "Organization Test");
+
+			EVPNavigator.assertActiveBreadcrumb(activeItem = "Payment Confirmation");
+
+			for (var fieldName : list "Request ID,EVP Payment Date,Payment Value,Upload File") {
+				EVPRequest.verifyFieldsName(key_field = ${fieldName});
+			}
+
+			EVPRequest.confirmPayment(
+				dateFormat = "MMddYYYY",
+				dateRange = 0,
+				document = "Document_1.pdf",
+				paymentDate = "true",
+				paymentValue = 100);
+		}
+
+		task ("Then he can see that the status change to 'Closed'") {
+			EVPNavigator.goToNavItem(navItem = "Request");
+
+			EVPUtils.viewStatus(
+				organizationName = "Organization Test",
+				status = "Closed");
+		}
+
+		task ("When he goes to Request details page") {
+			EVPRequest.goToRequestOption(
+				key_option = "Details",
+				organizationName = "Organization Test");
+		}
+
+		task ("Then he can asset the Payment Confirmation date in details page") {
+			EVPRequest.viewRequestDetails(paymentDate = ${paymentDate});
+		}
+	}
+
+	@description = "LPS-164742 - The uploaded files are being stored on DM"
+	@priority = 3
+	test VerifyUploadedFileOnDM {
+		task ("Given I have a created request in 'Awaiting Payment Confirmation' status") {
+			for (var userApproval : list "Manager,Finance") {
+				User.logoutAndLoginPG(
+					userLoginEmailAddress = "${userApproval}@liferay.com",
+					userLoginFullName = "${userApproval} User");
+
+				EVPUtils.gotoEVP();
+
+				EVPNavigator.goToNavItem(navItem = "Request");
+
+				EVPRequest.goToRequestOption(
+					key_option = "Review",
+					organizationName = "Organization Test");
+
+				EVPOrganization.changeStatus(
+					button = "Approve",
+					typeMessage = "Your organization has been approved!");
+			}
+		}
+
+		task ("When the Finance user do the Payment Confirmation") {
+			EVPRequest.goToRequestOption(
+				key_option = "Pay",
+				organizationName = "Organization Test");
+
+			EVPNavigator.assertActiveBreadcrumb(activeItem = "Payment Confirmation");
+
+			for (var fieldName : list "Request ID,EVP Payment Date,Payment Value,Upload File") {
+				EVPRequest.verifyFieldsName(key_field = ${fieldName});
+			}
+
+			EVPRequest.confirmPayment(
+				dateFormat = "MMddYYYY",
+				dateRange = 0,
+				document = "Document_1.pdf",
+				paymentDate = "true",
+				paymentValue = 100);
+		}
+
+		ProductMenu.gotoPortlet(
```

Comment:
Hey @MGabriellaS, you need to log in with the user Test Test to then go to Portlet, because this test failed at this step.